geocoder_tester/base.py

```diff
@@ -133,11 +137,55 @@ def reverse_params(self, center, **kwargs):
                         .format(kwargs['detail']))
         return params
 
+class PeliasApi(GenericApi):
+    """ Access proxy for Pelias APIs. The API URL must be the v1 base URL without
+        the /search or /reverse path.
+    """
+
+    def search_params(self, query, **kwargs):
+        params = self._common_params(**kwargs)
+        params['text'] = query
+        if kwargs.get('center'):
+            params['focus.point.lat'], params['focus.point.lon'] = kwargs['center']
+
+        return params
+
+    def search_url(self):
+        return CONFIG['API_URL'] + '/search'
+
+    def reverse_url(self):
+        return CONFIG['API_URL'] + '/reverse'
+
+    def reverse_params(self, center, **kwargs):
+        params = self._common_params(**kwargs)
+        params['point.lat'], params['point.lon'] = center
+        if 'detail' in kwargs:
+            if kwargs['detail'] == 'street' or kwargs['detail'] == 'house':
+                params['layers'] = 'address'
```

Comment:
From what I read in the API documentation, I would expect:
* street => street
* house => address

And it might be worth to simply do the others as well for completeness:
* country => country
* state => region
* county => county
* city => locality
* district => neighbourhood

Reply:
Added missing mappings